pipeline.yaml

```diff
@@ -98,7 +98,7 @@ extends:
         -BASE_API_URL "https://$(internalAPIMUrl)/pts-pet-checker/v1/api"
         -ACCOUNT_MANAGEMENT_URL "https://your-account.defra.gov.uk/management/account-management"
         -NODE_ENV  "production"
-        -DEFRA_ID_REDIRECT_URI "https://$(environment)-check-a-pet-from-gb-to-ni.azure.defra.cloud/signin-oidc"
+        -DEFRA_ID_REDIRECT_URI "https://$(environment)-check-a-pet-from-gb-to-ni.service.gov.uk/signin-oidc"
```

Comment:
remove environment and the dash after. service stats with check

Reply:
updated it now as mentioned above